oracle/oracle_client.go

```diff
@@ -67,7 +67,7 @@ func (hr *HTTPRequester) Get(api, data string) []byte {
 
 // OracleClient is a temporary replacement fot the real oracle. its gets accurate results from a server.
 type OracleClient struct {
-	world  uint64
+	world  int
```

Comment:
why do you care?

Reply:
There is a bug in the Cobra package. On int it doesn't happen.

Reply:
just cast to int when needed

Reply:
Fixed